app/models/neighborhood.rb

```diff
@@ -1,102 +1,95 @@
 class Neighborhood
 
   DISTRICTS = {
-    castro: {
+    castro: OpenStruct.new({
       latitude: "37.7617",
       longitude: "-122.4351",
       name: "Castro"
-    },
-    civic_center: {
+    }),
+    civic_center: OpenStruct.new({
       latitude: "37.7793",
       longitude: "-122.4175",
       name: "Civic Center"
-    },
-    financial_district: {
+    }),
+    financial_district: OpenStruct.new({
       latitude: "37.7952",
       longitude: "-122.4029",
       name: "Financial District"
-    },
-    fishermans_wharf: {
+    }),
+    fishermans_wharf: OpenStruct.new({
       latitude: "37.8083",
       longitude: "-122.4156",
       name: "Fisherman's Wharf"
-    },
-    haight_ashbury: {
+    }),
+    haight_ashbury: OpenStruct.new({
       latitude: "37.7700",
       longitude: "-122.4469",
       name: "Haight-Ashbury"
-    },
-    hayes_valley: {
+    }),
+    hayes_valley: OpenStruct.new({
       latitude: "37.7730",
       longitude: "-122.4283",
       name: "Hayes Valley"
-    },
-    marina: {
+    }),
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.